outputs/root_gauges.json

```diff
@@ -127,7 +126,6 @@
     "80LUMIN-20rETH-arbitrum-root-8f44": "0x8f44a8cfb7fe682295fa663348060533732f437c",
     "DUSD-4POOL-BPT-arbitrum-root-91ce": "0x91ceeb8d46428c5b8d76debc8156992e45d2d63f",
     "B-50wstETH-50COW-gnosis-root-93ae": "0x93ae6971f03ce890fa4e9274ab441477b84dae5f",
-    "ECLP-GHO-GYD-arbitrum-root-9434": "0x9434c3955920a1fed5e28ebccef80dcd54dea180",
```

Comment:
became replaced by a new one, somehow the new one isnt picked up as a root gauge yet in the old gauges subgraph

Reply:
i made a note here: https://github.com/BalancerMaxis/bal_tools/issues/39